addons/im_livechat/static/src/embed/common/chatbot/chatbot_model.js

```diff
@@ -118,19 +118,20 @@ export class Chatbot extends Record {
         } else {
```

Comment:
What is this if/else exactly? If we are at last step, we trigger, and then we might receive more steps, but what do we do with them after pushing them at the end of the "if"?

Reply:
`if/else` is just there to ensure we have the next step, either by fetching it or because we already have it. Then it depends of the step. If it expects an answer we wait for user input otherwise we trigger the next step and so on until the end of the script.